README.md

```diff
@@ -16,14 +21,47 @@ ORGANIZATION_CATALOG_URL=http://localhost:8080/organizations
 CONCEPT_HARVESTER_URL=http://localhost:8080/concepts
 DATASET_HARVESTER_URL=http://localhost:8080/datasets
 INFORMATIONMODELS_HARVESTER_URL=http://localhost:8080/informationmodels
-DATASERVICE_HARVESTER_URL=http://localhost:8080/apis
+DATASERVICE_HARVESTER_URL=http://localhost:8080/dataservices
+FDK_BASE=http://localhost:8080
+METADATA_QUALITY_ASSESSMENT_SERVICE_HOST=http://localhost:8080/assessment
+SEARCH_FULLTEXT_HOST=http://localhost:8080/search
 ```
+### Running the application 
+#### in commandline 
 
 ```
 % docker-comopose up -d                             # start mockserver
 % pipenv shell                                      # open a session in the virtual environment
 % FLASK_APP=src FLASK_ENV=development flask run     # run application
 ```
+
+
+#### in pycharm
```

Comment:
Eg er ikkje kjempe-tilhengar av å skrive README for spesifikke IDEs, men orkar ikkje å ta ein fight på det. Prinsipielt skal README vere så "verkøty-agnostisk" som mulig.